test/hotspot/gtest/runtime/test_os_windows.cpp

```diff
@@ -722,6 +722,104 @@ TEST_VM(os_windows, processor_count) {
   }
 }
 
+TEST_VM(os_windows, large_page_init_multiple_sizes) {
+    // Enable large pages and set conditions for the test
```

Comment:
Consider switching to 2-space indentation to match the rest of the file.

Reply:
done